### PR TITLE
Long description test

### LINKED ISSUE #4168: Start TE: Add long description (character limit test)
Sign in
Start time entry
Enter description that exceeds the 2000 character character limit.
Does the error show successfully?
Does it prevent you from saving the entry?

### PR DESCRIPTION
**Pull request by heytherewill**
# What's this?
This adds a test to ensure that something goes wrong when adding huge descriptions

### Relationships
Closes #4168

## How is it done?
Add a massive chunk of text and then check for errors

## Review hints
Apparently we don't have anything preventing the user from actually saving the time entry (which IMO is completely fine, since this check is super fiddly and backend is doing it anyways), but just in case we want client side validation I'm pinging @maritoggl here

For reviewers: The UI takes a long while to enter that much text on iOS (almost a full minute) and it freezes (for about 25 seconds) on Android, so bear this in mind when testing.

## :squid: Permissions
Only me

### CODE REVIEW

**Comment by daividssilverio** [comment]
Based on what was said internally, I guess this test isn't needed.

File: Toggl.Tests.UI/StartTimeEntryTests.cs [hunk 1]
```diff
@@ -38,6 +41,24 @@ public void TappingTheDoneButtonCreatesANewTimeEntryWhoseDescriptionMatchesWhatW
...
         }
 
+        [Test]
+        public void CreatingATimeEntryWithTooManyCharactersInTheDescriptionMakesTheEntryErrorWhenSyncing()
+        {
+            var description = Enumerable
+                .Range(0, 3001)
+                .Select(_ => "a")
+                .Aggregate(new StringBuilder(), (builder, character) => builder.Append(character))
+                .ToString();
```
Comment (amulware) [inline Toggl.Tests.UI/StartTimeEntryTests.cs]: I mean... `new String('a', 3001)`?

Comment (heytherewill) [reply Toggl.Tests.UI/StartTimeEntryTests.cs]: NOT FUNCTIONAL ENOUGH

**Comment by heytherewill** [comment]
As discussed internally, this is no longer needed

**Pull request closed without merging**